jetty9/src/main/resources/loading_pages/default.loading.page.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<!--
+  ~ Copyright 2018 ThoughtWorks, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<html>
+<head>
+  <title>GoCD server - Loading ...</title>
+  <meta charset="utf-8">
+  <style type="text/css">
+    body {
+      margin: 0 auto;
+    }
+
+    #logo {
+      width: 120px;
+      margin: 2em auto;
+    }
+
+    #logo svg .st0 {
+      fill: black;
+    }
+
+    .section-by-status {
+      text-align: center;
+      font-size: 24px;
+      font-family: sans-serif, serif;
+      width: 50%;
+      margin: 0 auto;
+    }
+
+    .section-by-status a {
+      text-decoration: none;
+    }
+
+    #update-check-message {
+      text-align: center;
+      font-size: small;
+      margin-top: 2em;
+    }
+  </style>
+</head>
+
+<body>
+
+<div id="logo">
+  <!-- Copy of: server/webapp/WEB-INF/rails.new/app/assets/images/go_logo.svg -->
```

Comment:
Server is not up. Can't really depend on loading any assets, etc.

Reply:
Make sense.